### PR TITLE
html/css and linters added

### PR DESCRIPTION
**Pull request by marcokumu**
-Added HTML and CSS Linters.
-Added HTML code for the mobile skeleton.
-Also added CSS code for the mobile skeleton

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <a class="okumu" href="">Okumu Mark</a>
+
+        <i class="fa fa-bars"></i>
+      </nav>
+    </header>
+    <div>
```
Comment (AnselemOdims) [inline index.html:22]: - [ ] Please replace this `div` with a `section` tag inother to make it more semantic.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me for more.
+      </p>
+    </div>
+
+    <div class="socials">
+      <i class="fa fa-github"></i>&ensp; <i class="fa fa-linkedin"></i>&ensp;
+      <i class="fa fa-twitter"></i>&ensp; <i class="fa fa-instagram"></i>&ensp;
+      <i class="fa fa-telegram"></i>
+    </div>
```
Comment (AnselemOdims) [inline index.html:37]: - [ ] Please kindly make sure the list of icons is grouped with a `ul` tag, and `li` per icon; each icon has a link (`a` tag)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <a class="okumu" href="">Okumu Mark</a>
+
+        <i class="fa fa-bars"></i>
+      </nav>
```
Comment (AnselemOdims) [inline index.html:20]: - [ ] Please kindly ensure that the toolbar section sits at the top of the page. At the moment, it is not exactly at the top
- [ ] Also, the padding does not meet the requirement of the original design.

### Your Design
image

### Original Design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me for more.
+      </p>
+    </div>
+
+    <div class="socials">
+      <i class="fa fa-github"></i>&ensp; <i class="fa fa-linkedin"></i>&ensp;
+      <i class="fa fa-twitter"></i>&ensp; <i class="fa fa-instagram"></i>&ensp;
+      <i class="fa fa-telegram"></i>
```
Comment (AnselemOdims) [inline index.html:36]: - Please kindly ensure to use the exact icons from the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design. You can export it from figma and save it to your project.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    <div>
+      <h1>Hey There. I'm Okumu</h1>
+      <h2>I'm a Software Developer</h2>
+
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me for more.
+      </p>
+    </div>
```
Comment (AnselemOdims) [inline index.html:31]: - [ ] At the moment, there is a vertical scroll on your page. Please try to remove that as it creates a bad user experience

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @marcokumu , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 4 above)
Comment (marcokumu) [reply index.html:36]: -About the icons; two of the icons are not really familiar to me( 'peace sign' icon and 'M' icon). So I thought putting social links that I actually have would be a viable option. 
-Or should I stick to the figma design STRICTLY. @AnselemOdims

File: index.html (hunk 5 above)
Comment (marcokumu) [reply index.html:31]: Successfully completed.

**Review by CollinsTatang — APPROVED** [review]
Hi @marcokumu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**